### ISSUE TITLE
RefNotes 2020-08-10 breaks my wiki

### ISSUE THREAD

**Issue by FilSalustri** [dwp-forge/refnotes#55]
I'm running Greebo.
I just tried to update RefNotes, but doing so causes a HTTP 500 error.
I don't know what other information I can provide to help you diagnose the problem.
Please let me know if I can do something on my end to help.

**Comment by goodnewz** [comment]
Here is the exact error out of Nginx. It looks like a missing class in bibtex.php:

`FastCGI sent in stderr: "PHP message: PHP Fatal error:  Class 'dokuwiki\Parsing\Parser' not found in /var/www/dokuwiki/lib
/plugins/refnotes/bibtex.php on line 11" while reading response header from upstream, client: XXX.XXX.XXX.XXX., server: xxx.com, request: "GET / HTTP/2.0", upstream: "fastcgi://unix:/var/run/php/php7.2-fpm.sock:"`

**Comment by dwp-forge** [comment]
Duplicate of #54.

**Comment by FilSalustri** [comment]
To get the wiki working again, I had deleted the 2020-08-10 version of RefNotes.
I have downloaded and manually installed the old version, which seems to work.
The permissions and file ownerships are consistent with all other plugins I have.

**However**, I do NOT see the config panel for RefNotes in the Configuration Manager, so I cannot reconfigure it as it was.
I have looked in all the obvious (to me) places for info, but can't find anything.

I'm hoping someone can point me at the instructions for completing a manual installation such that the config panel appears on the Configuration Manager.
